style.css

```diff
@@ -878,6 +878,209 @@ span {
   margin-bottom: 8px;
 }
 
+.desktop {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(157, 157, 157, 0.5);
+  pointer-events: auto;
+  position: fixed;
+  height: 100%;
+  top: 0;
+  left: 100rem;
+  width: 100%;
+  z-index: 100;
+}
+
+.desktop.active {
+  pointer-events: auto;
+  left: 0;
+}
+
+.desktop__area {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  height: 90%;
+  max-width: 68rem;
+  width: 100%;
+}
+
+.desktop__area-big-image {
```

Comment:
- [ ] [Optional] For better UX try to remove the transition on the hover over the ```See Project```

![Screenshot_20220519_060725](https://user-images.githubusercontent.com/25227022/169212998-85dbd2fe-3247-4db1-b47c-d7a125b1fe57.png)
![Screenshot_20220519_060729](https://user-images.githubusercontent.com/25227022/169213013-9a50f552-fa15-4c68-a895-0e8ae0bf2219.png)